src/libumf.c

```diff
@@ -39,12 +45,26 @@ void umfTearDown(void) {
         umfMemspaceLowestLatencyDestroy();
         umfDestroyTopology();
 #endif
+
+        if (utils_is_running_in_proxy_lib_with_size_threshold()) {
```

Comment:
IMO this name is too long. I would prefer is_proxy_lib && threshold > 0

Reply:
`threshold` is not available here (in `libumf.c`). It can be replaced with something like:
```c
if (utils_is_running_in_proxy_lib() && utils_is_running_with_size_threshold()) {
}
```
Is it really better?